### DISCUSSION TITLE
Filter by email suject

### DISCUSSION THREAD

**Discussion by devopsmash** [postalserver/postal#2019] [Help with using Postal] [answered]
There is any way to filter emails by email subject or part of it?

I can't find it here
![image](https://user-images.githubusercontent.com/22152786/173224103-a9ed58b1-3972-4c3f-a842-240322501672.png)

**Comment by devopsmash** [comment, marked as answer]
It seems that there is a workaround [here](https://github.com/postalserver/postal/discussions/1779)

Do you prefer to convert this ticket to a feature request or to close it?

**Reply by willpower232** [reply]
I don't think the situation has changed since that discussion so it is up to someone to implement it I guess.
